readme/manual-setup.md

```diff
@@ -2,6 +2,31 @@
 
 Welcome to the Ozone FOSS manual setup guide. This guide details the setup of Ozone FOSS step by step.
 
+### Destroy the running instance containers
+If you have already set up Ozone before you may need to clean up your local environment first:
+
+<table>
+<tr>
+<td> macOS </td> <td> Linux </td>
+</tr>
+<tr>
+<td>
+
+```bash
+$ ./destroy-demo.sh
+```
+
+</td>
+<td>
+
+```bash
+$ sudo -E ./destroy-demo.sh
+```
+
+</td>
+</tr>
+</table>
+
```

Comment:
I have reworded this, but now we need to put it in the right place which is neither at the very beginning nor at the very end.